src/entry-point-static/build.tsx

```diff
@@ -0,0 +1,23 @@
+import fs from "fs";
+import path from "path";
+
+import generatePageContent, { PageInfo } from "./generatePageContent";
+
+let pageInfos: PageInfo[] = [
+  { title: "Recidiviz COVID-19 Dashboard", location: "/" },
+  { title: "Test Page", location: "/test" },
+];
+
+console.log("Rendering React statically into HTML files:");
+
+for (let pageInfo of pageInfos) {
+  let dirPath = path.join("dist", "." + pageInfo.location);
```

Comment:
Small question: why the concatenation instead of just another comma to separate the period and `pageInfo.location` into separate args?

Reply:
Thanks for the callout - that works!

```js
> const path = require("path")
> path.join("dist", "/test")
'dist/test'
```

I was thinking of Python's `os.path.join`, which behaves differently:

```py
>>> import path from os
>>> path.join("dist", "/test")
'/test'
```